library/Ivoz/Ast/Domain/Model/PsEndpoint/PsEndpointInterface.php

```diff
@@ -154,6 +154,4 @@ public function getResidentialDevice(): ?ResidentialDeviceInterface;
     public function setRetailAccount(?RetailAccountInterface $retailAccount = null): static;
 
     public function getRetailAccount(): ?RetailAccountInterface;
-
-    public function isInitialized(): bool;
```

Comment:
Why is this gone? 100 times...

Reply:
This is because of the regenerated entities process after ivoz-core update.